draft-irtf-cfrg-frost.md

```diff
@@ -216,13 +216,10 @@ The following terminology is used throughout this document.
 * `MIN_SIGNERS` denotes the threshold number of participants required to issue a signature, where MIN_SIGNERS <= MAX_SIGNERS.
 * `NUM_SIGNERS` denotes the number of signers that participate in an invocation of FROST signing, where
   MIN_SIGNERS <= NUM_SIGNERS <= MAX_SIGNERS.
-* An identifier is an integer value associated with a participant, or signer,
-  and is a value in the range [1, MAX_SIGNERS].
+* The `Identifier` type is a `Scalar` restricted to the values [1, MAX_SIGNERS] associated with a participant, or signer.
```

Comment:
@chris-wood is this the right place/way to do this? We define the `Element` and `Scalar` types below in the context of group operations, so we haven't defined `Scalar` yet by this point in the doc